nemo_text_processing/text_normalization/hi/data/whitelist/alternatives.tsv

```diff
@@ -0,0 +1,45 @@
+Hon.	Honorable
```

Comment:
Since these are English, are they necessary for HI TN?

Reply:
There's likely a fair amount of codeswitching even in native Hindi text. So this may be valid. However, probably the cleaner way is to stick with only Hindi and have a separate PR for codeswitched text.

Reply:
I’m unsure how this file fits into the overall grammar. Should it be retained, or would it be better to remove it?

Reply:
echoing Travis's suggestion about separate PR for code-switched TN. Let's keep only Hindi content. Check if the file is used in your whitelist grammar. If not, delete; if it is, change for Hindi terms only